CRM/Report/Form/Contribute/Detail.php

```diff
@@ -882,8 +882,8 @@ public function sectionTotals() {
       foreach (array_merge($sectionAliases, $this->_selectAliases) as $alias) {
         $ifnulls[] = "ifnull($alias, '') as $alias";
       }
-      $this->_select = "SELECT " . implode(", ", $ifnulls);
-      $this->_select = CRM_Contact_BAO_Query::appendAnyValueToSelect($ifnulls, $sectionAliases);
+      $select = "SELECT " . implode(", ", $ifnulls);
+      $select = CRM_Contact_BAO_Query::appendAnyValueToSelect($ifnulls, $sectionAliases);
```

Comment:
@seamuslee001 it looks like you are setting a variable and then immediately overwriting it. Can L885 just be deleted?

Reply:
I hadn't noticed that but yes I believe your right